### PR TITLE
Adds board games with grads event

### PR DESCRIPTION
**Pull request by claricen**

### CODE REVIEW

File: content/S2016/S16-event-boardgames.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+title: Board Games Night with Grads
+Date: 2016-5-12
+Category: Events
+Tags: social
```
Comment (ehashman) [inline content/S2016/S16-event-boardgames.md]: Maybe add an 'academic' tag.

**Comment by ehashman** [comment]
@claricen, can you please fix this PR so it's a single commit? You appear to have merged two identical branches into each other! (Check out [this graph](https://github.com/wics-uw/website/network) for a pictoral representation.)

**Comment by claricen** [comment]
Squished the commits

File: content/S2016/S16-event-boardgames.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+title: Board Games Night with Grads
+Date: 2016-5-12
+Category: Events
+Tags: social, academic
+Slug: board-games-s16
+Author: Clare Ng
+Summary: Come out for a night of board games with grad students!
+
+Join us for a night of board games with some female computer science
```
Comment (ehashman) [inline content/S2016/S16-event-boardgames.md]: This looks good to me on the Heroku deploy, I just had a question about this line. Did the WiCS grad committee specifically give us this description or are we making an assumption about the gender identities of the grad students that will be present? If there are e.g. non-binary grad students attending I think this description would make them uncomfortable.

Comment (claricen) [inline content/S2016/S16-event-boardgames.md]: No, that was a mistake on my part - what should I change it to?

Comment (ehashman) [inline content/S2016/S16-event-boardgames.md]: You can probably just remove "female" and it will be fine!

**Pull request merged**